tests/ml/pytorch/test_pytorch_model_upload_pytest.py

```diff
@@ -112,12 +112,23 @@ def download_model_and_start_deployment(tmp_dir, quantize, model_id, task):
 
 
 class TestPytorchModel:
+    @property
+    def quantize(self) -> bool:
+        # quantization does not work on ARM processors
```

Comment:
Can the `quantize()` fixture on line 87 be removed now?

Reply:
Actually, my property can be removed now. Quentin fixed it in the mean time using fixtures so I'll keep his solution.